.github/workflows/nightly_cleanup.yml

```diff
@@ -11,7 +11,8 @@ env:
   AWS_PROFILE: "infex"
   # remember to also update tests.yml!
   AWS_REGION: "eu-west-2"
-  CLEANUP_NEWER_THAN: "6 hours"
+  CLEANUP_NEWER_THAN: "6h"
+  # TODO: renovate this
   CLOUD_NUKE_VERSION: "v0.35.0"
```

Comment:
example on how I did it in c8-multi-region:
https://github.com/camunda/c8-multi-region/blob/b35492abe1039112cd547d31eac99ff912e90661/.github/workflows/nightly_aws_region_cleanup.yml#L50-L56